packages/AlgernonTests.package/ALGTestAction.class/instance/testDoIt.st

```diff
@@ -0,0 +1,7 @@
+testing
+testDoIt
```

Comment:
I don't know if this is your test, but nethertheless, this should be moved to on of the  ALGTest Subclasses.
(Because we don't want to test search behaviour and initialise ALGCores outside of ALGTest, as the cleanup procedures etc. are located there)

Additionally I have the feeling, that we already have a test for doit 🤔.

Reply:
I think the test got deleted/moved, but this PR will add it again, because its starting point is so old. Please just remove it. It seems like a merge went wrong or something.